.github/workflows/build-and-test.yml

```
@@ -0,0 +1,57 @@
+name: Build and Test
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+  push:
+    branches-ignore:
+      - main
+concurrency:
+  group: build-and-test-${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    env:
+      BOT_GH_TOKEN: ${{ secrets.BOT_GH_TOKEN }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16.19.1'
```

저희 레포에서 node 18 이상으로 사용하라고 하고 있어서, 18 혹은 더 높은 버전으로 지정하면 좋을 것 같습니다!

```suggestion
          node-version: '18'
```